### DISCUSSION TITLE
WHY this is Happening

### DISCUSSION THREAD

**Discussion by ayesparshh** [Cyfrin/foundry-full-course-cu#2076] [Q&A] [answered]
on running this command source .env 

this is the output its not loading correctly.
SAPOLIA_RPC_URL: command not found
PRIVATE_KEY: command not found
RPC_URL: command not found

![Screenshot 2024-07-22 170720](https://github.com/user-attachments/assets/89d47839-5ed4-4659-9fac-ddd95374749a)

**Comment by ayesparshh** [comment, marked as answer]
solved

**Reply by EngrPips** [reply]
Awesome, I'm glad you figured this out. Keep crushing it.
